documentation/Functions.md

```diff
@@ -622,19 +603,19 @@ Note: The title should never be empty and it is of type string.
     - **Required**
     - **Type**- Array of JSON objects with files, content, or images to be rendered within the Katacoda step. 
     - **Description**-This function consists of three attributes. The use for this function is to display an image and some descriptive text. No Katacoda syntax is allowed in                         the files or the content!
-        * 1st attribute i.e.  "file": Path to a file whose content is to be displayed in the Katacoda step (e.g. .asciidoc or .txt file). The file should be following the                                formating of asciidoc files. 
-        * 2nd attribute i.e. "content": Plain text to be displayed in the Katacoda step. This Text should be following the formating of asciidoc files.
-        * 3rd attribute i.e. "image": Path to an image to be displayed in the Katacoda step. It should be placed under subfolder of the playbook directory. 
+        * First attribute i.e.  "file": Path to a file whose content is to be displayed in the Katacoda step (e.g. .asciidoc or .txt file). The file should be following the                                formating of asciidoc files. 
```

Comment:
This is not consistent in the whole document yet. Sometimes you use `First attribute: ...` and sometimes `First attribute i.e. 
 ...`. I would prefer the first one

Reply:
Modified to first one.